src/transactions/test/FeeBumpTransactionTests.cpp

```diff
@@ -49,7 +49,7 @@ feeBumpUnsigned(TestAccount& feeSource, TestAccount& source, TestAccount& dest,
 
 static TransactionFrameBasePtr
 feeBump(Hash const& networkID, TestAccount& feeSource, TestAccount& source,
-        TestAccount& dest, int64_t outerFee, int64_t innerFee, int64_t amount)
+        TestAccount& dest, int64_t outerFee, uint32_t innerFee, int64_t amount)
```

Comment:
maybe keeping `int64_t` for fees is fine, but just assert here that `innerFee` doesn't overflow `uint32_t`.
Right now you've done `static_cast` at the call sites without an assert, and it's quite error prone.